src/app/layout.tsx

```diff
@@ -1,16 +1,31 @@
+"use client"
```

Comment:
What does this do?

Reply:
Thats a react server components thing, since React released server components now on Next you have to define that when you want to have access to browser apis and assume render on the client basically.

https://adhithiravi.medium.com/what-are-server-components-and-client-components-in-react-18-and-next-js-13-6f869c0c66b0